<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReelKeeper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="1,3">
      <item>
       <widget class="QFrame" name="leftArea">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QPushButton" name="helpButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>75</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Google Sans</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>–ò–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è üÜò</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="libButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>75</horstretch>
               <verstretch>75</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>75</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Google Sans</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>–ë–∏–±–ª–∏–æ—Ç–µ–∫–∞ üìö</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="settingsButton">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>75</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Google Sans</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>–ù–∞—Å—Ç—Ä–æ–π–∫–∏ üõ†Ô∏è</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="reelkeeperLogo">
             <property name="font">
              <font>
               <family>Google Sans Medium</family>
               <pointsize>28</pointsize>
              </font>
             </property>
             <property name="text">
              <string> üéûÔ∏èReelKeeper</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="rightArea">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="infoLayout">
           <property name="spacing">
            <number>60</number>
           </property>
           <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
            <widget class="QLabel" name="infoLogo">
             <property name="font">
              <font>
               <family>Google Sans Medium</family>
               <pointsize>48</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="ReelKeeper - —ç—Ç–æ —É–¥–æ–±–Ω–æ–µ –ø—Ä–∏–ª–æ–∂–µ–Ω–∏–µ, –∫–æ—Ç–æ—Ä–æ–µ –ø–æ–º–æ–∂–µ—Ç –≤–∞–º –æ—Ä–≥–∞–Ω–∏–∑–æ–≤–∞—Ç—å –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—é –æ –ø—Ä–æ—Å–º–æ—Ç—Ä–µ–Ω–Ω—ã—Ö —Ñ–∏–ª—å–º–∞—Ö –∏ —Å–µ—Ä–∏–∞–ª–∞—Ö."/>
             </property>
             <property name="text">
              <string>üéûÔ∏èReelKeeper</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Google Sans Medium</family>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ReelKeeper - —ç—Ç–æ —É–¥–æ–±–Ω–æ–µ –ø—Ä–∏–ª–æ–∂–µ–Ω–∏–µ, –∫–æ—Ç–æ—Ä–æ–µ –ø–æ–º–æ–∂–µ—Ç –≤–∞–º&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;–æ—Ä–≥–∞–Ω–∏–∑–æ–≤–∞—Ç—å –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—é –æ –ø—Ä–æ—Å–º–æ—Ç—Ä–µ–Ω–Ω—ã—Ö —Ñ–∏–ª—å–º–∞—Ö –∏ —Å–µ—Ä–∏–∞–ª–∞—Ö.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;–°–æ–∑–¥–∞—Ç–µ–ª—å: Evgenij Fisko&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/venterum/ReelKeeper&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline; color:#0000ff;&quot;&gt;–†–µ–ø–æ–∑–∏—Ç–æ—Ä–∏–π ReelKeeper –Ω–∞ GitHub&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
